### PR TITLE
feat: add jumpstart llama-2 notebooks

### PR DESCRIPTION
**Pull request by ulrichkr**
*Issue #, if available:*

*Description of changes:* Add JumpStart Llama-2 inference notebooks.

*Testing done:* Run notebooks end-to-end.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x ] I have tested my notebook(s) and ensured it runs end-to-end
- [x ] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb [hunk 1]
```diff
@@ -0,0 +1,383 @@
+{
```
Comment (vivekmadan2) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>How about run llama chat assistant models in sagemaker jumpstart </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (vivekmadan2) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>This seems to belong in the technical section. How about just highlighting the fact that by using this model, you accept the license </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (vivekmadan2) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>How about putting notes at the end of the notebook?</p><p> </p><p>Also, how about input format before everything else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (vivekmadan2) [inline introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>Do you want add the default instance type and max token information clearly. May be some information on how to change default instance type as in falcon or just pointing to jumpstart model documentation </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (ulrichkr) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>Planning to change to these:</p><ol><li>Text completion: Run Llama 2 models in SageMaker JumpStart</li><li>Chat completion: Run Llama 2 models in SageMaker JumpStart</li></ol>

Comment (ulrichkr) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>Added a section on  changing instance type. The total number of tokens is described in the notes section below.</p>

Comment (ulrichkr) [reply introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb:1]: <p>We decided to keep notes next to supported parameters. Moved input format description up.</p>

**Pull request merged**